### PR TITLE
[APSI ]Updated APSI to 0.9.0

### PR DESCRIPTION
**Pull request by WeiDaiWD**
**Describe the pull request**

- #### What does your PR fix?
Updated `ports/apsi` to version 0.9.0.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
x86-windows-static, x64-windows-static, arm64-windows-static, x64-linux, arm64-linux, arm64-osx, x64-osx

No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/apsi/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "apsi",
-  "version-semver": "0.8.2",
+  "version-semver": "0.9.0",
   "port-version": 1,
```
Comment (JonLiu1993) [inline ports/apsi/vcpkg.json:4]:
```suggestion
```

Comment (JonLiu1993) [reply ports/apsi/vcpkg.json:4]: When we update port version, we should remove "port-version".

File: ports/apsi/portfile.cmake [hunk 2]
```diff
@@ -3,11 +3,9 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO microsoft/APSI
-    REF ba71aeb28a9f21e4ad59c45aa88232b099ce0b87 #0.8.2
-    SHA512 810bcbe0afa3d1c9d299a85bc4266135bdf9adc33bfc754c59731f6cfa6a89d449fb134cef34c4614742bd50e9f8f3916e5b64998dcea69883ca27b7da3c5f04
+    REF 2dff8dcd39c361527ea3b320f87cb8e71dd4f777 #0.9.0
+    SHA512 16c52642719f1d67dfaa70d963ba8795ac618f250752a1f95d91d4b1db8b51b2598999dcc9a9a7a3dbe8537943a3c3bf2ec684cd2697fca88135b01009961213
     HEAD_REF main
-    PATCHES
-        fix-find_package.patch
```
Comment (JonLiu1993) [inline ports/apsi/portfile.cmake:10]: @WeiDaiWD, thanks for your pr, please delete the patch files under the port folder and submit again.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/apsi/portfile.cmake (hunk 2 above)
Comment (WeiDaiWD) [reply ports/apsi/portfile.cmake:10]: Done.

File: ports/apsi/vcpkg.json (hunk 1 above)
Comment (WeiDaiWD) [reply ports/apsi/vcpkg.json]: This is removed.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Comment by autoantwort** [comment]
Hm since this update the build fails with 
```
CMake Error at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/scripts/buildsystems/vcpkg.cmake:843 (_find_package):
  Could not find a configuration file for package "SEAL" that is compatible
  with requested version "4".

  The following configuration files were considered but not accepted:

    /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/arm64-osx/share/seal/SEALConfig.cmake, version: 4.1.1

Call Stack (most recent call first):
  CMakeLists.txt:120 (find_package)


```

**Comment by autoantwort** [comment]
I wounder how the pipeline succeeded at all. For me the build now fails on all operating systems.

**Comment by autoantwort** [comment]
Ah the last pipeline probably run before 0a77b03b49a33b46af1bca5c6546b3c6c0625362 was merged. It is also a baseline regression: https://github.com/microsoft/vcpkg/runs/10897043328

**Comment by BillyONeal** [comment]
https://github.com/microsoft/vcpkg/pull/29204/

**Pull request merged**